Packages/Core/Editor/Generator/Generator.cs

```diff
@@ -58,7 +58,11 @@ public void Generate(string type, string baseWritePath, bool isEquatable, List<A
             var templateSearchPath = Runtime.IsUnityAtomsRepo ?
                 Directory.GetParent(baseWritePath).FullName : // "Packages"
                 Directory.GetParent(Application.dataPath).FullName;
+#if UNITY_2019_1_OR_NEWER
```

Comment:
This check is not necessary. You should always be able to do: 
`var templatePaths = Directory.GetFiles(templateSearchPath, "UA_Template*.txt", SearchOption.AllDirectories);`